src/modules/coreHttpApi/openapi.yml

```diff
@@ -431,6 +442,146 @@ paths:
+                successorContent:
+                  type: object
+                  additionalProperties: false
+                  required:
+                    - value
+                  properties:
+                    value:
+                      $ref: "#/components/schemas/AttributeValue"
+                    tags:
+                      $ref: "#/components/schemas/IdentityAttributeContent_tags"
+                    validFrom:
+                      $ref: "#/components/schemas/AttributeContent_validFrom"
+                    validTo:
+                      $ref: "#/components/schemas/AttributeContent_validTo"
+
+      responses:
+        200:
+          description: |
+            Success. Succeeding a Relationship Attribute will return an
+            additional ID of the notification sent to the peer.
+          content:
+            application/json:
+              schema:
+                type: object
+                additionalProperties: false
+                required:
+                  - result
+                properties:
+                  result:
+                    type: object
+                    additionalProperties: false
+                    required:
+                      - predecessor
+                      - successor
+                    properties:
+                      predecessor:
+                        $ref: "#/components/schemas/Attribute"
+                      successor:
+                        $ref: "#/components/schemas/Attribute"
+                      notificationId:
+                        $ref: "#/components/schemas/NotificationID"
+
+          headers:
+            X-Response-Duration-ms:
+              schema:
+                $ref: "#/components/schemas/HeaderContent_X-Response-Duration-ms"
+            X-Response-Time:
+              schema:
+                $ref: "#/components/schemas/HeaderContent_X-Response-Time"
+        401:
+          $ref: "#/components/responses/Unauthorized"
+        403:
+          $ref: "#/components/responses/Forbidden"
+        404:
+          $ref: "#/components/responses/NotFound"
+
+  /api/v2/Attributes/{attributeId}/NotifyPeer:
+    post:
+      operationId: notifyPeerAboutRepositoryAttributeSuccession
+      description: |
+        Notifies a peer about a succeeded and previously shared repository
+        attribute.
+      tags:
+        - Attributes
+      parameters:
+        - name: attributeId
+          in: path
+          required: true
+          schema:
+            $ref: "#/components/schemas/AttributeID"
+          description: ID of the succeeded repository attribute
+      requestBody:
+        content:
+          application/json:
+            schema:
+              additionalProperties: false
+              properties:
+                peer:
+                  $ref: "#/components/schemas/Address"
+
+      responses:
+        200:
+          description: |
+            "Success. Returns the successor and predecessor own shared identity
```

Comment:
Own shared identity attributes is mentioned here, would it be better to use repository attributes?